test/backups.test.js

```diff
@@ -25,29 +25,104 @@ describe('backups', () => {
     this.application = null;
   });
 
-  const BASE_ITEM_COUNT = 4;
+  const BASE_ITEM_COUNT_ENCRYPTED = [
+    'Note',
```

Comment:
This is kind of weird. Here you're limiting all tests in this file to only deal with 2 notes. The idea with `BASE` is that these are the number of items that the system is creating that your test doesn't know about, so if you're counting items, make sure you account for these.

Reply:
I don't think tests in this page should be limited to creating only 2 notes, and should be able to choose at their discretion how many notes to create. If they choose to create 10 notes, they can, and they should be able to ascertain that there are 10 + BASE items total.

Reply:
You're right, this is making too many assumptions about the tests in this file

Reply:
Are you saying the distinction between encrypted and decrypted base item count is weird?

Reply:
> Are you also saying the distinction between encrypted and decrypted base item count is weird?

No, wasn't looking at that